resources/secretsmanager_secrets.go

```diff
@@ -0,0 +1,247 @@
+				Description: "The last date and time that this secret was modified in any way.",
+				Type:        schema.TypeTimestamp,
+			},
+			{
+				Name:        "last_rotated_date",
+				Description: "The most recent date and time that the Secrets Manager rotation process was successfully completed",
+				Type:        schema.TypeTimestamp,
+			},
+			{
+				Name:        "name",
+				Description: "The friendly name of the secret",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "owning_service",
+				Description: "Returns the name of the service that created the secret.",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "primary_region",
+				Description: "The Region where Secrets Manager originated the secret.",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "rotation_enabled",
+				Description: "Indicates whether automatic, scheduled rotation is enabled for this secret.",
+				Type:        schema.TypeBool,
+			},
+			{
+				Name:        "rotation_lambda_arn",
+				Description: "The ARN of an Amazon Web Services Lambda function invoked by Secrets Manager to rotate and expire the secret either automatically per the schedule or manually by a call to RotateSecret.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("RotationLambdaARN"),
+			},
+			{
+				Name:        "rotation_rules_automatically_after_days",
+				Description: "Specifies the number of days between automatic scheduled rotations of the secret",
+				Type:        schema.TypeBigInt,
+				Resolver:    schema.PathResolver("RotationRules.AutomaticallyAfterDays"),
+			},
+			{
+				Name:        "secret_versions_to_stages",
+				Description: "A list of all of the currently assigned SecretVersionStage staging labels and the SecretVersionId attached to each one",
+				Type:        schema.TypeJSON,
+			},
+			{
+				Name:        "tags",
+				Description: "The list of user-defined tags associated with the secret",
+				Type:        schema.TypeJSON,
+				Resolver:    resolveSecretsmanagerSecretsTags,
+			},
+		},
+	}
+}
+
+// ====================================================================================================================
+//                                               Table Resolver Functions
+// ====================================================================================================================
+
+func fetchSecretsmanagerSecrets(ctx context.Context, meta schema.ClientMeta, _ *schema.Resource, res chan interface{}) error {
+	c := meta.(*client.Client)
+	svc := c.Services().SecretsManager
+	cfg := secretsmanager.ListSecretsInput{}
+	for {
+		response, err := svc.ListSecrets(ctx, &cfg, func(options *secretsmanager.Options) {
+			options.Region = c.Region
+		})
+		if err != nil {
+			return err
+		}
+
+		// get more details about the secret
+		for _, n := range response.SecretList {
+
+			cfg := secretsmanager.DescribeSecretInput{
+				SecretId: n.ARN,
```

Comment:
I think you are missing a call to `ListSecretVersionIds` to get all of the different versions